services/repository-managers/src/main/java/org/sagebionetworks/repo/manager/DockerManagerImpl.java

```diff
@@ -230,4 +246,35 @@ public void dockerRegistryNotification(DockerRegistryEventList registryEvents) {
 			}
 		}
 	}
+
+	@Override
+	public void addDockerCommitToUnmanagedRespository(UserInfo userInfo, String entityId,
+			DockerCommit commit) {
+		ValidateArgument.required(entityId, "entityId");
+		ValidateArgument.required(commit.getDigest(), "digest");
+		if (dockerNodeDao.getRepositoryNameForEntityId(entityId)!=null) {
+			throw new IllegalArgumentException("Cannot add a commit for a managed Docker repository.");
```

Comment:
clarify that commit can in fact be added for managed repo, but only through the Docker client